### PR TITLE
Set Arbitrum Fees to 0

### PR DESCRIPTION
**Pull request by ElliotFriedman**

### CODE REVIEW

File: contracts/test/integration/vip/vip6.sol [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        proposal[0].description = "Set mint fee to 0 basis points on DAI PSM";
+
+        proposal[1].target = ArbitrumAddresses.VOLT_DAI_PSM;
+        proposal[1].value = 0;
+        proposal[1].arguments = abi.encodeWithSignature(
+            "setRedeemFee(uint256)",
+            0
+        );
+        proposal[1]
+            .description = "Set redeem fee to 0 basis points on USDC PSM";
```
Comment (k-xo) [inline contracts/test/integration/vip/vip6.sol:67]: This should be "Set redeem fee to 0 basis points on DAI PSM"

**Review by k-xo — COMMENTED** [review]
Looking good, 1 question though.

Should the PR creation of the DAI PSM on mainnet be renamed to VIP-7 I figure this PR will be carried out first.

File: contracts/test/integration/vip/vip6.sol (hunk 1 above)
Comment (ElliotFriedman) [reply contracts/test/integration/vip/vip6.sol:67]: yes, but we don't need to change the description.

**Review by k-xo — APPROVED** [review]
LGTM

**Review by k-xo — APPROVED** [review]
LGTM

**Review by Namaskar-1F64F — APPROVED** [review]
Looks good to me. Validated the `testProposalArbitrum` test calldata as well. 

Instead of changing the inherited `vip` in the `Runner` and commenting out what type of network, maybe you could abstract it out a little more and have a new integration test file for each VIP that inherits the `Runner` contract, and testing could be with the VIP itself, e.g. `testVIP6`.

**Comment by ElliotFriedman** [comment]
> Instead of changing the inherited `vip` in the `Runner` and commenting out what type of network, maybe you could abstract it out a little more and have a new integration test file for each VIP that inherits the `Runner` contract, and testing could be with the VIP itself, e.g. `testVIP6`.

Directionally, this feels correct, the issue with this is that we want the test runner to only run active proposals, and doing this design would mean that we would have to constantly be changing the value in the package.json on where proposals should run from so that we don't run past proposals, or we would need to continually move VIP files to another directory. So all solutions to this problem have tradeoffs that I just don't think make sense to make.

**Review by RusseII — APPROVED** [review]

**Review by thomas-waite — APPROVED** [review]
Looks good to me. Simple and clear

**Pull request merged**